src/_P101_WakeOnLan.ino

```diff
@@ -298,10 +297,9 @@ boolean Plugin_101(uint8_t function, struct EventStruct *event, String& string)
       if (validTaskIndex(event->TaskIndex) &&
           (cmd.equalsIgnoreCase(F(CMD_NAME_P101)) ||
            cmd.equalsIgnoreCase(getTaskDeviceName(event->TaskIndex)))) {
-        taskEnable = Settings.TaskDeviceEnabled[event->TaskIndex];
 
         // Do not process WOL command if plugin disabled. This code is for errant situations which may never occur.
-        if (!taskEnable) {
+        if (!Settings.TaskDeviceEnabled[event->TaskIndex]) {
```

Comment:
This is 'old fashion'/legacy code, as `PLUGIN_WRITE` is never called if the device isn't enabled.